### ISSUE TITLE
Bad Interpreter Path

### ISSUE THREAD

**Issue by eknowlton** [dbcli/pgcli#1011]
## Description
I've installed pgcli with pip using `pip install pgcli`. I then went to use pgcli and received the error:

```
$  pgcli messaging
zsh: /usr/local/bin/pgcli: bad interpreter: /usr/local/opt/python/bin/python3.6: no such file or directory
```

It seems the stub created for pgcli points to a specific directory for the python interpreter...

```
$ cat /usr/local/bin/pgcli
#!/usr/local/opt/python/bin/python3.6

# -*- coding: utf-8 -*-
import re
import sys

from pgcli.main import cli

if __name__ == '__main__':
    sys.argv[0] = re.sub(r'(-script\.pyw?|\.exe)?$', '', sys.argv[0])
    sys.exit(cli())
```

## Your environment
I'm currently running OS X 10.14.3 with python installed using `asdf-python`. I have python 2.7 and python 3.7 installed.

```
$ pip freeze
...
pgcli==2.0.2
...
```

I love these command line tools and am currently using pgcli with a workaround by simply changing this line to use `#!/usr/bin/env python3`

**Comment by amjith** [comment]
This seems to be a mismatch between the pip and the default python on your system. Can you try this? 

```
python -m pip install pgcli 
```

That will use the pip from the current python. 

Or you can always use brew to install pgcli. For that you'll have to use `brew install pgcli`

**Comment by eknowlton** [comment]
Thanks for the response. It seems this would fix it and seems to be a common way to mitigate a issue like this. 

I have also realized I believe I had `pgcli` installed before I installed python using `asdf` and I actually think this might have been the cause. Possibly used brew to install python or another python version manager.

Either way thanks for the response and your awesome work on this project!
